mu4/appshell/qml/DevTools/Audio/SynthSettings.qml

```diff
@@ -0,0 +1,143 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.3
+import MuseScore.UiComponents 1.0
+import MuseScore.Midi 1.0
+
+Rectangle {
+
+    color: ui.theme.backgroundColor
+
+    SynthsSettingsModel {
+        id: settingsModel
+    }
+
+    Component.onCompleted: {
+        settingsModel.load()
+    }
+
+    Item {
+        id: synthListPanel
+
+        property int currentIndex: 0
+
+        anchors.top: parent.top
```

Comment:
if qml component created used Component:
```
Component {
        id: synthSettingsComp
        SynthSettings {}
    }
```
It will be created without a parent, and a warning will be in log: "TypeError: Cannot read property of null". It's better to write: 
```
anchors.top: (parent ? parent.top : 0)
```
When a parent will be set, it will apply changes.

Reply:
In this case all right, no any warning output.